### ISSUE TITLE
Trouble Installing Plugin

### ISSUE THREAD

**Issue by pioneerathletics** [oroskodias/m2-fix-cache-topmenu#1]
Hi,
I'm having trouble installing this plugin. After successfully running setup:upgrade I attempt to run setup:di:compile but the installation gets stuck at 57%. 
```
Compilation was started.
Interceptors generation... 4/7 [=======>-----]  57% 19 secs 230.0 MiB

```
If you could provide installation instructions, I would be grateful.

**Comment by orozcodiaz** [comment]
@pioneerathletics just seen your request, my friend. Can you provide more information about your issue? What's the spec of your environment and what version of Magento you're running?

**Comment by pioneerathletics** [comment]
Hi  @oroskodias ,

Thanks for responding.

Here is the Magento environment:

Magento 2.3.3 (updated from 2.1)
Development site in Developer Mode
PHP 7.1
Redis Cache enabled

So I could be wrong on how to install this but, traditionally, I’ve created a folder AWS/TopMenuCacheFix in the app/code folder. I then run setup:upgrade and then setup:di:compile. When I run the compilation command I get stuck at 4/7 and it doesn’t continue:
`php magento setup:di:compile
Compilation was started.
Interceptors generation... 4/7 [========>-----]  57% 15 secs 230.0 MiB`

**Comment by orozcodiaz** [comment]
@pioneerathletics you don't have to run `setup:di:compile` in developer mode. Just clean the contents of `generated/` folder and you'll be able to see results straight after you enter your website. I will try to reproduce the issue in a couple of hours. Let me know if you have installed some other 3rd party extensions.

**Comment by orozcodiaz** [comment]
@pioneerathletics Thank you for reporting the issue. Just fixed it in the version 1.0.1 (it's in master now). Please also make sure you use `Aws` instead of `AWS`
